New web player interface

quevedo opened the pull request:
- Added new web services for playlist information and control
- New theme
- Replaced pure with bootstrap
- Moved javascript to a separate static file

quevedo committed:
New web player interface

- Added new web services for playlist information and control
- New theme
- Replaced pure with bootstrap
- Moved javascript to a separate static file
- Add remove functionality

equalsraf commented on app.py:
```diff
@@ -116,6 +116,15 @@ def addurl():
 
+@app.route('/now', methods=['POST'])
+@needsmpd
+def now():
```
Why is this called `now()`? Why not `playid` like the mpd method?

equalsraf commented on app.py:
```diff
@@ -132,6 +141,13 @@ def add(ytid):
+@needsmpd
+def previous():
+    """Go to the previous song."""
+    g.client.previous ()
```
avoid the extra space here `previous()` instead of `previous ()`

equalsraf commented on app.py:
```diff
@@ -146,6 +176,12 @@ def stop():
 
+@app.route('/list')
+@needsmpd
+def list():
```
Would prefer playlistinfo() instead of just info().

quevedo committed:
Add some color-based feedback

-Change the color of items after clicking them and while awaiting for a result (success or error)
-Remove duplicated function for closing error message

equalsraf commented:
Merged and deployed in production. Thanks.
